examples/tools/scene_viewer.rs

```diff
@@ -178,11 +179,20 @@ fn keyboard_animation_control(
             }
         }
 
-        if keyboard_input.just_pressed(KeyCode::Return) {
+        if *changing {
+            // change the animation the frame after return was pressed
             *current_animation = (*current_animation + 1) % scene_handle.animations.len();
             player
                 .play(scene_handle.animations[*current_animation].clone_weak())
                 .repeat();
+            *changing = false;
+        }
+
+        if keyboard_input.just_pressed(KeyCode::Return) {
```

Comment:
This should probably be added in the controls guide message

Reply:
it is already?

Reply:
🤦‍♂️ oh right, it's the enter key, nevermind